examples/boxes3d.rs

```diff
@@ -116,8 +116,7 @@ fn initialise_camera(world: &mut World) {
             rotation: Quaternion::one(),
             scale: Vector3::from_value(1.),
             translation: Vector3::new(0., 0., 1.),
-        })
-        .with(GlobalTransform::default())
+        }).with(GlobalTransform::default())
```

Comment:
rustfmt

Reply:
opardum:amethyst-rhusics simon$ cargo fmt --version
rustfmt 0.99.1-stable (da17b689 2018-08-04)
opardum:amethyst-rhusics simon$ rustup update stable
info: syncing channel updates for 'stable-x86_64-apple-darwin'

  stable-x86_64-apple-darwin unchanged - rustc 1.29.2 (17a9dc751 2018-10-05)

Sorry to burst your bubble mate.